core/src/main/scala/org/apache/spark/input/WholeTextFileInputFormat.scala

```diff
@@ -48,11 +50,11 @@ private[spark] class WholeTextFileInputFormat
    * Allow minPartitions set by end-user in order to keep compatibility with old Hadoop API,
    * which is set through setMaxSplitSize
    */
-  def setMinPartitions(context: JobContext, minPartitions: Int) {
+  def setMinPartitions(sc: SparkContext, context: JobContext, minPartitions: Int) {
```

Comment:
Please update the above comment to explain the new behavior.

Reply:
Ok,thanks